deps/triemap/triemap.c

```diff
@@ -1065,6 +1066,82 @@ int TrieMapIterator_NextContains(TrieMapIterator *it, char **ptr, tm_len_t *len,
   return 0;
 }
 
+int TrieMapIterator_NextWildcard(TrieMapIterator *it, char **ptr, tm_len_t *len, void **value) {
+  if (TimedOut_WithCounter(&it->timeout, &it->timeoutCounter)) {
+    return 0;
+  }
+
+  while (array_len(it->stack) > 0) {
```

Comment:
Shouldn't we check for timeout inside the loop?

Reply:
Good point.